options.c

```diff
@@ -160,6 +166,16 @@ void apply_options(struct bm_menu *menu) {
 #endif
 	bm_menu_set_panel_overlap(menu, !no_overlap);
 	bm_menu_set_monitor(menu, get_monitor_idx(monitor));
+#ifdef HAVE_BEMENU_PASSWORD_INDICATOR
+	if (!password || !strcmp(password, "hide"))
+		bm_menu_set_password(menu, BM_PASSWORD_HIDE);
+	else if (!strcmp(password, "none"))
+		bm_menu_set_password(menu, BM_PASSWORD_NONE);
+	else if (!strcmp(password, "indicator"))
+		bm_menu_set_password(menu, BM_PASSWORD_INDICATOR);
```

Comment:
Please also a fallback to `BM_PASSWORD_HIDE`,
Other than that it looks great.

Reply:
`-x foo`.

Reply:
Ah, right. Good catch! Should be fixed now.